src/Tizen.NUI.Components/Controls/RecyclerView/Layouter/GridLayouter.cs

```diff
@@ -509,7 +518,7 @@ public override void NotifyItemInserted(IItemSource source, int startIndex)
             // Insert Single item.
             if (source == null) throw new ArgumentNullException(nameof(source));
             if (colView == null) return;
-            if (isSourceEmpty)
+            if (isSourceEmpty || StepCandidate == 0)
```

Comment:
I cannot imagine s.t. `isSourceEmpty` is false, but `StepCandidate` is 0. (Since we make StepCanditate as 1 at line 280) Is this branch is right way to determine empty group case?

Reply:
StepCandidate is for normal item candidate.
if source has a group but chlidren is empty, isSourceEmpty is false but children is empty so StepCandidate should be 0.
and NotifyItemInserted is called, we don't have stepCandidate, so we need to re-intialize to find out this source has Children or not and re-calculate StepCandidate.